docs/dev/connectors/cassandra.zh.md

```diff
@@ -122,7 +122,7 @@ CREATE KEYSPACE IF NOT EXISTS example
     WITH replication = {'class': 'SimpleStrategy', 'replication_factor': '1'};
 CREATE TABLE IF NOT EXISTS example.wordcount (
     word text,
-    count bigint,
+    `count` bigint,
```

Comment:
I'm not sure about this. 

According to Cassandra docs, the `count` is a non-reserved keyword. And non reserved keywords doens't need single or double quotes. 

> Reserved keywords cannot be used as identifier, they are truly reserved for the language (but one can enclose a reserved keyword by double-quotes to use it as an identifier). Non-reserved keywords however only have a specific meaning in certain context but can used as identifier otherwise.

https://cassandra.apache.org/doc/latest/cql/appendices.html

Reply:
Yes.